tests/test_client.py

```diff
@@ -934,10 +967,10 @@ def create_modifier_list(self, num_records):
 
     def _create_item(self, start_date, num_records=1):
         mod_lists = self.get_all('modifier_lists', start_date)
-        if mod_lists:
-            mod_list_id = random.choice(mod_lists).get('id')
-        else:
-            mod_list_id = self.create_modifier_list(num_records).body.get('objects').get('id')
+        if not mod_lists:
+            mod_lists = self.create_modifier_list(1).body.get('objects')
```

Comment:
Why was this hard coded to 1 rather than using `num_records`? It might be fine, I just can't remember if we were intentionally trying to get 1 to 1 relationship between streams or if it doesn't matter.

Reply:
If you look at a few lines below you see that we always grab a random id from the list, so we only actually need 1 mod list to create N items. When create modifier list is called not in this context it will create however many are required.

Reply:
Also `self.create_modifier_list(num_records).body.get('objects').get('id')` was failing because `self.create_modifier_list(num_records).body.get('objects')` returns a list, not the first object's dictionary value. I think a previous refactor left this bad code in there from a past time.